src/SFA.DAS.RoatpGateway.Web.UnitTests/Services/OrganisationChecks/Orchestrator/OrganisationStatusTests.cs

```diff
@@ -9,14 +10,15 @@
 using SFA.DAS.RoatpGateway.Web.Infrastructure.ApiClients;
 using SFA.DAS.RoatpGateway.Web.Services;
 
-namespace SFA.DAS.AdminService.Web.Tests.Services.Gateway.OrganisationChecks.Orchestrator
+namespace SFA.DAS.RoatpGateway.Web.UnitTests.Services.OrganisationChecks.Orchestrator
 {
     [TestFixture]
     public class OrganisationStatusTests
     {
         private GatewayOrganisationChecksOrchestrator _orchestrator;
         private Mock<IRoatpApplicationApiClient> _applyApiClient;
         private Mock<ILogger<GatewayOrganisationChecksOrchestrator>> _logger;
+        private Mock<IRoatpApiClient> _charityApiClient;
```

Comment:
Rename the client to IOuterApiClient. This is the convention that is being followed. And rename the variable to _outerApiClient

Reply:
done